src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/ServerSentEventsTransport.cs

```diff
@@ -94,7 +94,7 @@ private async Task ProcessAsync(Uri url, HttpResponseMessage response)
         {
             // Start sending and polling (ask for binary if the server supports it)
             var receiving = ProcessEventStream(response, _transportCts.Token);
-            var sending = SendUtils.SendMessages(url, _application, _httpClient, _logger, _inputCts.Token);
+            var sending = SendUtils.SendMessages(url, _application!, _httpClient, _logger, _inputCts.Token);
```

Comment:
Consider Debug.Assert on _application.

Repeat for other usages in transports.